daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -739,11 +739,12 @@ object Converter {
 
   def toUser(v: SValue): Either[String, User] =
     v match {
-      case SRecord(_, _, vals) if vals.size == 2 =>
+      case SRecord(_, _, vals) if vals.size >= 2 =>
         for {
           id <- toUserId(vals.get(0))
           primaryParty <- toOptional(vals.get(1), toParty)
-        } yield User(id, primaryParty)
+          // TODO um-for-hub: Support isDeactivated and metadata
```

Comment:
could you expend `um-for-hub` and add a link to an github issue or a jira ticket.

Reply:
fyi, removed this TODO note from the code for now